app/controllers/api/v1/hostgroups_controller.rb

```diff
@@ -33,10 +33,10 @@ def show
         param :architecture_id, :number
         param :medium_id, :number
         param :ptable_id, :number
-        param :puppet_ca_proxy_id, :number
```

Comment:
These should get deprecated instead of removed entirely, you can pick the primary url if the user just gives the proxy ID